k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: probate-cron-reactivate-dormant-cases
+spec:
+  releaseName: probate-cron-reactivate-dormant-cases
+  chart:
+    git: git@github.com:hmcts/probate-cron
+    ref: master
+    path: probate-cron
+  values:
+    job:
+      image: hmctspublic.azurecr.io/probate/back-office:prod-7c60114-20220816141606 #{"$imagepolicy": "flux-system:demo-probate-back-office"}
```

Comment:
please remove the demo- from image policy

Reply:
Removed